agent/taskresource/cgroup/cgroup.go

```diff
@@ -22,11 +22,11 @@ import (
 	"sync"
 	"time"
 
-	"github.com/aws/amazon-ecs-agent/agent/api"
 	"github.com/aws/amazon-ecs-agent/agent/resources/cgroup"
 	"github.com/aws/amazon-ecs-agent/agent/utils/ioutilwrapper"
 	"github.com/cihub/seelog"
 	"github.com/containerd/cgroups"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
looks like we track a [revision](https://github.com/aws/amazon-ecs-agent/blob/master/agent/Gopkg.toml#L62) for this dependency. since they have [versioned releases](https://github.com/opencontainers/runtime-spec/releases), we can track one of those. although this could be a separate PR against dev i guess? but ya, we should do this.

Reply:
we were using this already in the cgroup codebase